app/services/katello/registration_manager.rb

```diff
@@ -243,9 +243,7 @@ def finalize_registration(host)
         host.subscription_facet.update_from_consumer_attributes(host.subscription_facet.candlepin_consumer.
             consumer_attributes.except(:guestIds, :facts))
         host.subscription_facet.save!
-        host.subscription_facet.update_subscription_status
-        host.content_facet.update_errata_status
-        host.refresh_global_status!
+        host.refresh_statuses(::Katello::HostStatusManager::STATUSES)
```

Comment:
`refresh_statuses` calls `refresh!` on every `relevant?` status, and then calls `refresh_global_status!`. So it seems like it's better just to do this.